uint32 marker_id
geometry_msgs/PoseStamped pose
